bibliopixel/drivers/__init__.py

```diff
@@ -0,0 +1 @@
+from . channel_order import ChannelOrder
```

Comment:
I actually want to get things out of these __init__.py

Putting things into an __init__.py is a tiny bit bad for development.  The reason is that anything there gets automatically imported under many circumstances, and in my experience, 80% of circular imports come from something accidentally imported in __init__.py.

All else being equal, the best __init__.py is empty - though there are specific reasons that you might need things in an __init__.py.